.github/workflows/visual-regression.yml

```diff
@@ -12,6 +12,7 @@ on:
       - '!assets/**/__tests__/**/*.js'
       - '!assets/**/test/*.js'
       - '!assets/**/*.test.js'
+      - 'stories/**'
```

Comment:
Let's also add a pattern for matching stories written in CSF which may not be in this directory.

Reply:
Makes sense @aaemnnosttv. I changed the pattern and you can see the VRTs are running!

Reply:
Thanks @danielgent, although the fact that they're running now can also be attributed to this path 😉 
https://github.com/google/site-kit-wp/blob/890e24fac0144fdcc08f4eb02c54c4170f34e757/.github/workflows/visual-regression.yml#L10

The new pattern should match them all though so +1 to this change 👍